save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/dto/GitRepoDto.kt

```diff
@@ -0,0 +1,19 @@
+package org.cqfn.save.preprocessor.dto
+
+import kotlinx.serialization.Serializable
+
+/**
+ * Data class with repository information
+ *
+ * @param url - url of repo
+ * @param username - username to credential
+ * @param password - password to credential
+ * @param branch - branch to clone
+ */
+@Serializable
+data class GitRepoDto(
```

Comment:
This should be moved to `save-common`, because it will be shared between backend and preprocessor